pkg/asset/installconfig/aws/basedomain.go

```diff
@@ -98,5 +119,9 @@ func GetPublicZone(name string) (*route53.HostedZone, error) {
 	if res == nil {
 		return nil, errors.Errorf("No public route53 zone found matching name %q", name)
 	}
+
+	publicZoneID := strings.TrimPrefix(aws.StringValue(res.Id), "/hostedzone/")
+	res.Id = &publicZoneID
```

Comment:
why this change? i think the users of the return value `route53.HostedZone` would expect the zone id to be in AWS documented form... ?

Reply:
I have no idea why I did this or what I was thinking at the time. Fixed!